aeon/schema/core.py

```diff
@@ -37,7 +37,7 @@ def subject_state(pattern):
     return {"SubjectState": _reader.Subject(f"{pattern}_SubjectState_*")}
 
 
-def messageLog(pattern):
+def message_log(pattern):
```

Comment:
Can we focus this PR only on the major naming changes and terminology? I know we disagree on best practices for casing in python, but if we could leave this out of this PR so we can focus on the core discussions on naming, that would help a lot.